flask_blogging/templates/blogging/base.html

```diff
@@ -1,15 +1,16 @@
-<!DOCTYPE html>
+
 <html>
 <head lang="en">
     <meta charset="UTF-8">
-    {% block style %}
+    {% block styles %}
     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" rel="stylesheet"/>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-markdown/2.8.0/css/bootstrap-markdown.min.css"
```

Comment:
This css is used only by the editor. That is why I had it in editor alone. Would that be compatible with flask-bootstrap in place? If not we can leave it in here.